Scripts/Insights_19.10.3_hotfix.ps1

```diff
@@ -0,0 +1,52 @@
+﻿param(
+   [Parameter(Mandatory=$true)][string]$username
+  ,[Parameter(Mandatory=$true)][string]$password
+)
+
+function Get-checkBuild($tenant)
+{
+        $cubeBuilding = 0
+        while ($cubeBuilding -ne 1)
+        {
+            $buildResult = & "C:\Program Files (x86)\UiPath\Orchestrator\Tools\UiPath.InsightsAdminTool.exe" buildStatus -t $tenant | out-string
+            $buildStatus = $buildResult.Split(“`n”)
+
+            if($buildStatus[1].StartsWith("Build successfully ended"))
+            {
+                $cubeBuilding = 1;
+            }
+            else
+            {
+                Write-Output "$tenant Cube is still building. Waiting..."
+                Start-Sleep -s 10
+            }
+        }
+
+}
+
+Write-Output "Copying .jar"
+Copy-Item test.txt -Destination "C:\Program Files\Sisense\DataConnectors\JVMContainer\Connectors\UiFrost\com.sisense.connectors.jdbc.UiFrost.jar" -Recurse -force
+
+
+Write-Output "Getting list of tenants"
+$listTenants = & "C:\Program Files (x86)\UiPath\Orchestrator\Tools\UiPath.InsightsAdminTool.exe" list -u $username -p $password | out-string
+$tenantArray = $listTenants.Split(“`n”)
+
+Write-Output "Starting cube rebuilds"
+foreach ($row in $tenantArray)
+{
+    if($row -match 'tenant (?<Tenant>.+) with insights ENABLED')
```

Comment:
I don't understand this can you add a comment to explain the check?

Reply:
{
    **#Iterate and find tenants that have Insights enabled and do a full rebuild**
    if($row -match 'tenant (?<Tenant>.+) with insights ENABLED')
    {        
        $tenant =   $Matches.Tenant        
        Get-checkBuild $tenant        
        Write-Output "Triggered $tenant cube rebuild"
        & "$insightsAdminToolPath\UiPath.InsightsAdminTool.exe" rebuild -t $tenant | out-null        
        Get-checkBuild $tenant        
        Write-Output "$tenant cube is rebuilt"
    }

![image](https://user-images.githubusercontent.com/820683/73787035-498a2480-474f-11ea-9efe-2ba415333b3a.png)